awx/main/tasks/system.py

```diff
@@ -667,7 +667,7 @@ def cluster_node_heartbeat(dispatch_time=None, worker_tasks=None):
             active_task_ids.extend(task_list)
 
         # Convert dispatch_time to datetime
-        ref_time = datetime.fromisoformat(dispatch_time) if dispatch_time else datetime.now()
+        ref_time = datetime.fromisoformat(dispatch_time) if dispatch_time else now()
```

Comment:
`now()` returns a timezone aware datetime object whereas the previous datetime.now() did not. Following where `ref_time` gets used, in the reaper:

```
jobs = UnifiedJob.objects.filter(base_Q & Q(started__lte=ref_time))
```

:+1:  One less warning of the form:

```
/var/lib/awx/venv/awx/lib64/python3.11/site-packages/django/db/models/fields/__init__.py:1595: RuntimeWarning: DateTimeField UnifiedJob.started received a naive datetime (2025-04-28 19:02:38.172021) while time zone support is active.
  warnings.warn(
```